frontend/apps/filemanager/filemanager.lua

```diff
@@ -146,6 +146,8 @@ function FileManager:setupLayout()
         return_arrow_propagation = true,
         -- allow Menu widget to delegate handling of some gestures to GestureManager
         filemanager = self,
+        -- let Menu widget to merge TitleBar's FocusManager layout
```

Comment:
Sorry, a few rewording to more correct English for easier future reading :)
As I understand it, so I may be wrong, correct me if my wording is not what you/the code means.
`-- let Menu widget merge our title_bar into its own TitleBar's FocusManager layout`

Reply:
Comment message updated